integration/presto/src/main/java/org/apache/carbondata/presto/PrestoCarbonVectorizedRecordReader.java

```diff
@@ -223,8 +223,6 @@ private void initBatch() {
     for (int i = 0; i < fields.length; i++) {
       if (queryModel.isDirectVectorFill()) {
         vectors[i] = new ColumnarVectorWrapperDirect(columnarBatch.column(i));
-      } else {
-        vectors[i] = new CarbonColumnVectorWrapper(columnarBatch.column(i), filteredRows);
```

Comment:
you can remove if check also I guess, directly prepare ColumnarVectorWrapperDirect and also add a comment with TODO that if presto implements row level filtering, need to prepare CarbonColumnVectorWrapper instead of ColumnarVectorWrapperDirect

Reply:
done